collect_repos.py

```diff
@@ -58,6 +65,11 @@ def run(username, token, organization, prefix, list, deadline):
 				get_repo(repo, newDir)
 		elif repo.name.startswith(prefix):
 			get_repo(repo, newDir)
+	
+	if no_match:
+		logger.error("\n\n\nCould not find these ccids in the student list:")
+		for ccid in no_match:
+			logger.error(ccid)
```

Comment:
awesome! Can you add to the readme that the script will say if it didn't find any ccids in the list so whoever is running it keeps an eye out on it